website/content/docs/installation.md

```diff
@@ -1,83 +1,195 @@
 +++
 description = "How to install Bloop on your machine"
-title = "The installation process"
+title = "Installation"
 date = "2018-02-08T16:35:00+01:00"
 draft = false
 weight = 1
 bref = "Here are all the instructions to install Bloop"
 toc = true
 +++
 
-### Instructions for Mac OS
+### Mac OS
 
-Installing Bloop with [Homebrew](https://brew.sh) is the easiest and fastest way to install Bloop on
-Mac OS.
-
-We publish an updated formula along with every release of Bloop. You can install it using:
+Install with [Homebrew](https://brew.sh) via:
 
 ```sh
 $ brew install scalacenter/bloop/bloop
 ```
 
-The formula will also take care of installing a property list (`plist`) that will take care of
-starting the Bloop server when you log onto your machine. This way, you don't have to think about
-starting it yourself.
+The formula installs a property list (`plist`) that starts the Bloop server
+automatically when you log onto your machine. This way, you don't have to
+start it manually.
 
 If you don't want the Bloop server to start automatically, you can disable it: 
 
 ```sh
 $ brew services stop bloop
 ```
 
-If you want to have it start automatically again, type:
+If you want to have it start automatically *again*, type:
 
 ```sh
 $ brew services start bloop
 ```
 
-Finally, Homebrew will also take care of automatically installing completions for zsh and Bash.
-
-Go to [Next steps]({{< ref "#next-steps" >}}) to configure Bloop and start using it.
+Homebrew also automatically installs completions for zsh and Bash.
 
-### Instructions for other platforms
+### Other platforms
 
 Install bloop in other platforms (Windows, Unix, \*bsd) via our python script:
 
 ```sh
 $ curl -L https://github.com/scalacenter/bloop/releases/download/v1.0.0-M5/install.py | python2
 ```
 
-<span class="label warning">Note</span> you need to replace the version in the URL
+<span class="label warning">Remember</span> to replace the version in the URL
 by the latest version, which is [![Latest version](https://index.scala-lang.org/scalacenter/bloop/sbt-bloop/latest.svg)](https://index.scala-lang.org/scalacenter/bloop/sbt-bloop).
 
-#### Command line completion
+The installation script will also install completions for zsh and Bash, but
+you need to set them up manually.
 
-The installation script will also install completions for zsh and Bash.
+##### Zsh completions
 
-##### Completion in zsh
-
-To get the command line completions with zsh, add the following in your `~/.zshrc`:
+Add the following to your `~/.zshrc`:
 
 ```sh
 autoload -U compinit
 fpath=($HOME/.bloop/zsh $fpath)
```

Comment:
I'm not sure this line is correct. I've just run the install script above, which has created `.bloop/{bloop,blp-coursier,blp-server}`, though not a file called `zsh`.

Reply:
Because you're still using the last version. M7, which is cut tonight, will have it!

Reply:
What installation script have you used? If it's the one in `releases`, then it's normal because the completions didn't exist in our last release :)

The one in `bin` will install them. This will do the trick:

```sh
$ bin/install.py -v 1.0.0-M6+53-10bd93aa -n ebe7ab230dede05840edfff71064d3cb880cac32
```

Reply:
I copy/pasted the `curl` command from a few lines above.